packages/invoices/src/OutboundShipment/api/operations.graphql

```diff
@@ -1,3 +1,55 @@
+fragment PartialLocation on LocationNode {
```

Comment:
I'm thinking I should remove these fragments and just inline them as we're not actually using the generated types